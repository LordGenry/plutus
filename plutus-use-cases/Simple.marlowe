contract Simple {
    when not true and false
        or 20 Ada >= 2 Ada * 3 Ada + 1 Ada
        and now < block 1000 before block 128 then end
}
