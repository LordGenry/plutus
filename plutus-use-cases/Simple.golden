[When (OrObs (AndObs (NotObs TrueObs) FalseObs) (AndObs (ValueGE (Value 20) (AddValue (MulValue (Value 2) (Value 3)) (Value 1))) (BelowTimeout 1000))) 128 Null Null]